homeassistant/components/evohome/climate.py

```diff
@@ -90,11 +90,14 @@
             zone.zoneId,
             zone.name,
         )
+        new_entity = EvoThermostat(broker, zone)
+        broker.entities = {zone.zoneId: new_entity}
```

Comment:
It's ok to store the entity_id in a shared container but not the entity.

Reply:
OK (BTW, is unique_id)